src/seedu/addressbook/data/person/Person.java

```diff
@@ -65,6 +65,10 @@ public UniqueTagList getTags() {
     public void setTags(UniqueTagList replacement) {
         tags.setTags(replacement);
     }
+    
+    public String getPrintableString() {
+        return Printer.getPrintableString(name.getPrintableString(), phone.getPrintableString(), address.getPrintableString(), email.getPrintableString());
```

Comment:
This doesn't look right. Shouldn't you be passing `Printable` objects e.g. `name`  instead of `name.getPrintableString()`?